manifest.json

```diff
@@ -544,6 +544,15 @@
         "packageType": "Image"
       }
     ],
+    "amazon/dotnet5.0-base": [
```

Comment:
SAM CLI will need to be aware of a dotnet5 image based runtime, so that this option gets listed in sam init, could you confirm that this is the case?

Reply:
Yes, .NET 5 based template is available in `sam init`.